### ISSUE TITLE
Add Export QEMU Arguments to File

### ISSUE THREAD

**Issue by conath** [utmapp/UTM#2283]
UTM is a nice starting grounds for people who are just getting into QEMU. 

Therefore it would be helpful to be able to **export a UTM VM‘s generated QEMU parameters to a text file**, so one can continue tinkering outside UTM. This is tangentially related to #2280.

**Comment by osy** [comment]
What about export to clipboard?

**Comment by conath** [comment]
Sure, that would be good. Then one can simply paste it into a text editor.

**Comment by dylanmc** [comment]
I discovered this feature, via this issue, by searching for "headless qemu UTM". I tried it out, but the command it exported didn't work.

First, the command issued an argument with a path to a non-exist terminal file/device:

```
qemu-system-aarch64: -chardev pipe,id=term0,path=/Users/<me>/Library/Group Containers/WDNLXAD4W8.com.utmapp.UTM/8169C503-2646-4317-BE65-7F931B70884F.terminal: Could not open '/Users/<me>/Library/Group Containers/WDNLXAD4W8.com.utmapp.UTM/8169C503-2646-4317-BE65-7F931B70884F.terminal': No such file or directory
```

I found that there was a pair of files ending in `.terminal.in` and `.out` in that directory, so I tried editing the command to refer to the `...terminal` prefix instead, and got this error:

```
qemu-system-aarch64: -chardev spicevmc,name=usbredir,id=usbredirchardev0: 'spicevmc' is not a valid char driver name
```

Thinking that perhaps the `qemu-system-aarch64` command UTM uses has added new drivers, I looked for the one run by UTM (by examining the output of `ps auwwx`, and found this one: `/Applications/UTM.app/Contents/Frameworks/qemu-aarch64-softmmu.framework/Versions/A/qemu-aarch64-softmmu`, but when I try to run that command, I get `zsh: exec format error`. At this point I decided to chime in here.
